source/parser.hera

```diff
@@ -3450,6 +3440,17 @@ CaseClause
       children: $0
     }
 
+PatternExpressionList
+  ( BindingPattern / ConditionFragment ):first ( _? Comma _? ( BindingPattern / ConditionFragment ) )*:rest ->
```

Comment:
It might make sense to make `BindingPattern / ConditionFragment` into a separate rule (`PatternExpression`?). Plausibly `ConditionFragment` could be renamed into this, unless we want to use it elsewhere.